tests/e2e/specs/activate-and-setup/complete-business-section.ts

```diff
@@ -1,37 +0,0 @@
-/**
```

Comment:
@samueljseay thanks for deleting these 👍 👍  and moving it to the tests, much nicer that way (it drove me a bit crazy, but clearly not crazy enough to change it :/ )